fastlane/metadata/android/de/full_description.txt

```diff
@@ -1 +1,3 @@
 <i>Passky</i> ist ein einfacher Passwort-Manager, der auf einer Zero-Trust-Architektur arbeitet. Das bedeutet, dass nur der Benutzer seine Passwörter entschlüsseln kann. So können Benutzer ihre Passwörter sicher auf jedem Server speichern. Das heißt, wenn ein Server, auf dem alle Passwörter gespeichert sind, gehackt wird, kann der Hacker die Passwörter nicht entschlüsseln und die Daten auf diesem Server sind für ihn nutzlos.
+
+AntiFeatures: NonFreeNet - Es stellt eine Verbindung zu Google-Servern her, um Website-Symbole abzurufen
```

Comment:
> AntiFeatures: NonFreeNet - Die App ruft Icons für Websites über Google-Server ab.

Reply:
Sorry if I messed it up, online translation :)

Reply:
No prob, it's understandable – just sounds a bit weird. That's why I jumped in, being a "native speaker" (why must I giggle about that term? :see_no_evil:)

Some background on the change: your online-translation is a bit ambiguous ("retrieving website symbols" could also refer to symbols shown on any page, like emoticons). While "icons for websites" is not perfect, it's closer to what is meant. I was not sure whether "FavIcons" would be too technical.